version/description

```diff
@@ -1,11 +1,7 @@
-v0.4.0
-Bump dependencies to align with kubernetes v1.19.1
-
+v0.5.0
 Features:
-* Bump device-plugin-manager and kubelet deps (#30)
-* Bump dependencies (#29)
-Bugs:
-
+* tier1, Change tests to get namespace from cluster
+* tier1, Change KUBECONFIG to be configurable in functest
```

Comment:
These are the commits; none of this is a feature.

Write something like 'enable macvtap to be tested as part of CNAO tier1' .